client/src/components/App.jsx

```diff
@@ -15,7 +15,15 @@ function App() {
         <Overview />
       </OverviewContextProvider>
       <RelatedProducts />
+      <br />
```

Comment:
I believe we can make this code more DRY by adding CSS margining in the Questions component/